src/xcode/ENA/ENA/Source/Views/ENSetting/DescriptionTableViewCell.swift

```diff
@@ -20,20 +20,18 @@ import UIKit
 
 class DescriptionTableViewCell: UITableViewCell, ConfigurableENSettingCell {
 	@IBOutlet weak var titleLabel: ENALabel!
-	@IBOutlet var textView1: UITextView!
-	@IBOutlet var textView2: UITextView!
-	@IBOutlet weak var textView3: UITextView!
+	@IBOutlet weak var label1: UILabel!
+	@IBOutlet weak var label2: UILabel!
+	@IBOutlet weak var label3: UILabel!
```

Comment:
Just a small things. Swift Format kicks out ```weak``` from outlets when it's ran on a file. That's why most if not all outlets in the codebase are without ```weak``` now. Perhaps you could ran Swift Format on this file or just remove it by hand, for the sake of consistency?